autosklearn/pipeline/components/data_preprocessing/minority_coalescense/minority_coalescer.py

```diff
@@ -38,6 +38,7 @@ def get_properties(dataset_properties=None):
                 'handles_classification': True,
                 'handles_multiclass': True,
                 'handles_multilabel': True,
+                'handles_multioutput': False,
```

Comment:
Hey, could you please let me know why this doesn't work for multi-output regression?